src/main/java/no/nav/pto/veilarbportefolje/oppfolging/OppfolgingAvsluttetService.java

```diff
@@ -34,6 +40,14 @@ public OppfolgingAvsluttetService(ArbeidslisteService arbeidslisteService,
     public void behandleKafkaMelding(String kafkaMelding) {
         final OppfolgingAvsluttetDTO dto = JsonUtils.fromJson(kafkaMelding, OppfolgingAvsluttetDTO.class);
         final AktoerId aktoerId = dto.getAktorId();
+
+        final ZonedDateTime startdato = oppfolgingRepository.hentStartdato(aktoerId);
+        final ZonedDateTime sluttdato = dto.getSluttdato();
+        if (startdato.isAfter(sluttdato)) {
```

Comment:
evt. så er det greit å ta en if(startdato != null && startdato.isAfter(sluttdato))

Reply:
Hadde det i utgangspunktet, men det er 13 rader i prod som ikke har startdato, så tenker å heller rydde i de og sette den kolonnen til NOT NULL